pixel/image.go

```diff
@@ -202,7 +202,7 @@ func (img Image[T]) Get(x, y int) T {
 		// Monochrome.
 		var c Monochrome
 		offset := index / 8
-		bits := index - (offset * 8)
+		bits := index % 8
```

Comment:
This code looks different, but it doesn't change behavior. It's just a simplification.